pom.xml

```diff
@@ -136,6 +136,32 @@
                             </execution>
                         </executions>
                     </plugin>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-enforcer-plugin</artifactId>
+                        <version>3.0.0-M3</version>
```

Comment:
I think you downgraded by mistake in your last changes, you originally had put the latest version:
```suggestion
                        <version>3.0.0</version>
```

Reply:
`ossindex-maven-enforcer-rules` version `3.1.0` do not support `maven-enforcer-plugin` version `3.0.0` yet, had to downgrade to `3.0.0-M3` :/

Reply:
😱 Oh no, they claim compatibility with 3+: https://sonatype.github.io/ossindex-maven/enforcer-rules/
What happens if you don't specify the version like in their example?

Reply:
Nothing really as far as we are concerned :D

Reply:
^^ But like which version does it gets when you mvn install?

Reply:
We have two things here:
* ossindex (`3.1.0`): gets the [rules](https://ossindex.sonatype.org/) to be applied (_i.e vulnerable dependencies_).
* enforcer plugin (`3.0.0-M3`): checks if we have any dependency reported by the rules; the build fails if it is the case with an error message.

All this is applied to the build step only, and nothing is packaged to the final artifact :)

Reply:
Oh, my bad! I didn't get your question at first glance! 
In their example, they must be using `dependencyManagement` for `maven-enforcer-plugin` too, and but it is not mentioned in their example.
In the end, a version must be specified _(directly or indirectly)_.

Reply:
> In the end, a version must be specified (directly or indirectly).

Alright that's the part I was not sure about, thanks for confirming! 👍